src/org/loklak/LoklakServer.java

```diff
@@ -104,25 +104,7 @@
 import org.loklak.api.p2p.HelloService;
 import org.loklak.api.p2p.PeersServlet;
 import org.loklak.api.p2p.PushServlet;
-import org.loklak.api.search.ConsoleService;
-import org.loklak.api.search.EventBriteCrawlerService;
-import org.loklak.api.search.GenericScraper;
-import org.loklak.api.search.GithubProfileScraper;
-import org.loklak.api.search.InstagramProfileScraper;
-import org.loklak.api.search.LocationWiseTimeService;
-import org.loklak.api.search.MeetupsCrawlerService;
-import org.loklak.api.search.QuoraProfileScraper;
-import org.loklak.api.search.RSSReaderService;
-import org.loklak.api.search.SearchServlet;
-import org.loklak.api.search.ShortlinkFromTweetServlet;
-import org.loklak.api.search.SuggestServlet;
-import org.loklak.api.search.TimeAndDateService;
-import org.loklak.api.search.TweetScraper;
-import org.loklak.api.search.UserServlet;
-import org.loklak.api.search.VideoUrlService;
-import org.loklak.api.search.WeiboUserInfo;
-import org.loklak.api.search.WikiGeoData;
-import org.loklak.api.search.WordpressCrawlerService;
+import org.loklak.api.search.*;
```

Comment:
No! We didn't do this for a reason. Please undo this change.

Reply:
Ok but May I know the reason?

Reply:
The main reason why we do not do this is future compatibility, if there's a package which says `a.b.c.*` and `x.y.z.*` both having an class of say `Person` then there are higher chances for errors and the compilers getting confused. Also using the `*` means you're increasing the lookup time for the compiler while compilation by a small time but the resulting byte code will remain the same anyway.

I agree that having a long list of the imports can be quite intimidating but good IDEs always collapse them and you do not have to worry about what you import either because if there's a class you're using which isn't found in the file as an import, the IDE allows you to press a shortcut and it'll auto import that for you.

Reply:
Thanks
Tip: This change was done be intellij itself :)